support-frontend/app/config/UserBenefitsApi.scala

```diff
@@ -0,0 +1,8 @@
+package config
+
+import com.typesafe.config.Config
+
+class UserBenefitsApi(config: Config) {
```

Comment:
What is this class for? It seems to be just the same as the UserBenefitsApiConfig case class.

Reply:
Removed in ec11d92a6ca7d2655d8390e269e0f90adc1d7068.